src/Surfnet/StepupMiddleware/ApiBundle/Identity/Projector/RaCandidateProjector.php

```diff
@@ -147,14 +253,57 @@ public function applyRegistrationAuthorityRetractedEvent(RegistrationAuthorityRe
             $event->identityInstitution,
             $event->nameId,
             $event->commonName,
-            $event->email
+            $event->email,
+            $event->raInstitution
```

Comment:
Is `raInstitution` a property of the RegistrationAuthorityRetractedEvent? In the version I've reviewed, it was not!

Reply:
This was fixed while working on the event replay